OWNERS

```diff
@@ -0,0 +1,12 @@
+# TODO(jlewi): We should probably have OWNERs files in subdirectories that 
+# list approvers for individual components (e.g. Seldon folks for Seldon component)
+approvers:
+  - aronchick
+  - jlewi
+  - texasmichelle
+reviewers:
+  - aronchick
+  - carmine
+  - jlewi
```

Comment:
Don't duplicate approvers in reviewers; (approvers are automatically reviewers). So remove aronchick, jlewi, and texasmichelle as reviewers

Reply:
Done